extracters/aata_data.py

```diff
@@ -34,6 +34,10 @@
 
 localIdentifier = None # TODO: aat:LocalIdentifier?
 legacyIdentifier = None # TODO: aat:LegacyIdentifier?
+isbn10Identifier = None # TODO: aat for isbn10?
```

Comment:
local is vocab.LocalNumber.
I don't think we need variantTitleIdentifier, but open to discussion.

ISBN: http://vocab.getty.edu/aat/300417443  -- use for both 10 and 13.
ISSN: http://vocab.getty.edu/aat/300417430

Reply:
Ah. I seem to have gotten the modeling wrong, then. If I understand this correctly, then *each* local ID needs to be `vocab.LocalNumber(content="XYZ")`, right? Similarly for ISBNs, and ISSNs. If that's the case, I can't instantiate these here, because they can't be singletons, but I similarly can't just have them be types, because some of them are parameterized (ISBNs need a type *and* URI). Any thoughts?